test/support/pages/Common.js

```diff
@@ -107,11 +107,12 @@ define(function (require) {
         var lastUrl = currentUrl;
         return self.tryForTime(defaultTimeout, function () {
           // give the app time to update the URL
-          return self.sleep(500)
+          return self.sleep(501)
```

Comment:
It helps when looking at the debug logging output to know that it's this sleep we're seeing.